kiwi/xml_description.py

```diff
@@ -108,6 +112,25 @@ def load(self):
             raise KiwiDescriptionInvalid(message)
         return self._parse()
 
+    def _get_validation_details(self):
+        """
+        Run jing process to validate description against the schema
+
+        Jing command provides detailed feedback in case of schema
+        validation failure
+        """
+        try:
+            cmd = Command.run(
+                ['jing', Defaults.get_schema_file(), self.description_xslt_processed.name],
```

Comment:
Yep, I though it was already done by Command, as I got a FileNotFoundError message while trying silly commands. But yep, now I checked and it is just the forwarded exception message from subprocess. Wouldn't it make sense to test if the command is available in Command.run and raise KiwiCommandNotFoundError (subclass of KiwiCommandError) exception or something like? This way we would just have to distinguish the given exception from Command and we would not modify the behavior of current KiwiCommandError try/catch bocks.

What you think?